src/log/broadcast_backend.cr

```diff
@@ -0,0 +1,33 @@
+# A backend that broadcast to others backends.
+# Each of the referenced backends may have a different severity level filter.
+#
+# When this backend level is set that level setting takes precedence
+# over the severity filter of each referenced backend.
```

Comment:
I don't quite follow what this sentence means.

The implementation looks like if `BroadcastBackend#level` is not nil, it overrides any levels configured for the child backends. Why is that so? It feels a bit surprising.

Reply:
It is to allow programatic override of the level from a Log instance. See `Log#level=`.